static-alloc/src/unsync/mod.rs

```diff
@@ -0,0 +1,85 @@
+//! This module defines a simple bump allocator.
+//! The allocator is not thread safe.
+
+mod node;
+mod allocation;
+
+use node::{AllocationError, Node, BumpError};
+use allocation::Allocation;
+
+use core::{
+    mem::ManuallyDrop,
+    ptr::{NonNull},
+    cell::Cell,
+};
+
+/// A `Bump` is a simple bump allocator, that draws
+/// it's memory from another allocator. Bump allocators
+/// can be chained together using [`Bump::chain`]. 
+pub struct Bump {
```

Comment:
I just realized that the `Node` in this module is the `Bump` in the other. Can we rename this to `Chain` and `Node` to `Bump`, in case we find a nice way to expose it?

Reply:
The reason the current `Node` was kept seperate is because it requires special deallocation. At the moment I'm not sure how to handle that correctly without something like the current `Bump`.

I do find it a good idea however to seperate them as much as possible; a `Chain` should build upon the core `Bump`.